source/common/modules/markdown-editor/plugins/render-mermaid.js

```diff
@@ -39,6 +38,109 @@ const mermaid = require('mermaid')
     let codeblock = [] // Holds a mermaid code block
     let currentCursorPosition = cm.getCursor('from').line
 
+    // we need to initialize the mermaid API here, since we can only get the
+    // CSS color variables by querying the computed style of created documents
+    // get theme colors and font
+    let mermaid_bg0   = getComputedStyle(document.querySelector(".CodeMirror")).getPropertyValue('--mermaid-bg0').replace(/\s+/g, '')
+    let mermaid_bg1   = getComputedStyle(document.querySelector(".CodeMirror")).getPropertyValue('--mermaid-bg1').replace(/\s+/g, '')
```

Comment:
Additionally, you might want to run the .CodeMirror query select once, and save it to a variable. This might make computation a little bit better (I don't know whether Chrome caches this value, though …)